frontend/components/Main/MainNeedsFunds.tsx

```diff
@@ -102,7 +102,7 @@ export const MainNeedsFunds = () => {
             <span className="font-weight-600">
               {`${serviceFundRequirements.eth} XDAI `}
             </span>
-            - for gas & trading balance.
+            - for trading balance.
```

Comment:
@oaksprout noticed this https://valory-workspace.slack.com/archives/C06HNEWREG3/p1716894164845809?thread_ts=1716893331.832559&cid=C06HNEWREG3
hope that's fine that I updated the copy

Reply:
Thanks, looks good